archive_updater.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019 Google LLC
```

Comment:
ah, we might have missed this in previously added files, but you'll want 2020 here.